docs/ui/Commandk.tsx

```diff
@@ -0,0 +1,108 @@
+import { siteConfig } from '@/lib/config';
+import { Button, Dialog } from '@/ui';
+import { Command, CommandGroup } from 'cmdk';
+import { allContentPages } from 'contentlayer/generated';
+import React from 'react';
+
+import { useRouter } from 'next/navigation';
+
+import { Search } from '@marigold/icons';
+
+export const CommandMenu = () => {
+  // Toggle the menu when ⌘K is pressed
+  const router = useRouter();
+  const [open, setOpen] = React.useState(false);
+  const [search, setSearch] = React.useState('');
+
+  React.useEffect(() => {
+    const handleKeyDown = (e: {
+      key: string;
+      metaKey: boolean;
+      ctrlKey: boolean;
+      preventDefault: () => void;
+    }) => {
+      if (e.key === 'k' && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault();
+        setOpen(open => !open);
+      }
+    };
+    document.addEventListener('keydown', handleKeyDown);
+    return () => document.removeEventListener('keydown', handleKeyDown);
+  }, []);
+
+  const groupedPages = siteConfig.navigation.map(({ name, slug }) => {
+    const items = allContentPages
+      .filter(page => page.slug.includes(slug))
+      .map(({ title, slug, order }) => ({ title, slug, order }));
+    //sort by order or alphabetticly
+    //sort by order if there is an order
+    //if there is no order sort alphabetticly
+    items.sort((a, b) => {
+      if (typeof a.order == 'number' && typeof b.order == 'number') {
```

Comment:
Use [strict comparison](https://www.telerik.com/blogs/indepth-understanding-strict-abstract-equality-operators-javascript#:~:text=In%20JavaScript%2C%20the%20abstract%20and,not%20of%20the%20same%20type.). 

Than it's ok from my side. Well done 👍